docs/source/model_design/general_research.rst

```diff
@@ -618,12 +618,52 @@ In **randomized experimental studies**, the process of randomising individuals (
 
-	This section
+Randomization is a critical aspect of RCTs. The goal behind randomizing study subjects to exposure groups is to ensure that any potential confounders, both known and unknown, between the exposure and outcome are evenly distributed between groups; in other words, to eliminate any association between the confounder and the exposure (a necessary condition for Confounding_). When equal distribution of confounding variables across exposure groups is achieved via randomization, there will be no bias in the measurement of the association between the exposure and outcome due to confounding. This means that there is no need to identify and control for potential confounding variables when conducting RCTs. This makes RCTs a favorable study design for relationships between exposures and outcomes that are not well known, because identification of potential confounders is not necessary apriori.
+
+RCTs are almost always longitudinal in nature in that study subjects are tracked over time following administration of the exposure or treatment. Selection of study participants are generally required to be at-risk for the outcome (note that the outcome may be *remission* from a condition, requiring study subjects to be infected at study enrollment).
+
+Potential Biases
+^^^^^^^^^^^^^^^^
+
+Although randomization minimizes the potential for confounding bias, it may still occur if randomization of study subjects to exposure groups does not achieve equal distribution of confounding variables across groups. This may occur due to random chance and is more likely to occur when the study size is small. For this reason, RCTs generally report the distribution of demographic and other variables across exposure groups.
+
+Another source of potential bias in RCTs arises from lack of blinding of exposure groups. For instance, if a study subject was aware that they were assigned to take a status quo version of a drug rather than a new formulation of a drug, they may be less likely to comply with the treatment algorithm than a study subject who knew they were taking the new formulation. Further, study administrators may differentially treat patients in either the experimental or control groups, which could differentially affect outcomes. For this reason, **single-blinded** (blinding of exposure groups to study subjects OR study administrators), or more preferentially **double-blinded** (blinding of exposure groups to study subjects AND study administrators) RCTs are less subject to bias than non-blinded RCTs. The **placebo effect** is an example of a phenomenon that can bias study results in the lack of blinding study participants to their treatment group.
+
```

Comment:
Thanks for introducing single-blinded, double-blinded, and non-blinded RCTs, it's good to put some experimental examples here.